sktime/transformations/series/window_summarizer.py

```diff
@@ -1,434 +1,11 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
-"""Extract features across (shifted) windows from provided series."""
+"""Implement transformers for summarizing a time series."""
```

Comment:
this file should import `WindowSummarizer` from `series.summarize`, or all old imports will break (like the one in the docstring example)